inc/html.class.php

```diff
@@ -6517,6 +6517,8 @@ public static function compileScss($args) {
       }
 
       $scss->addImportPath(GLPI_ROOT);
+      $scss->addImportPath(GLPI_ROOT."/css");
+      $scss->addImportPath(GLPI_ROOT."/node_modules");
```

Comment:
This one is not OK. Indeed, required scss files are copied in `css/lib` directory to permit recompilation when using a production build of GLPI. I guess the callback declared below in `$scss->addImportPath()` have to be fixed. If some scss files are missing, they have to be added in webpack config section related to assets copy.

Reply:
i need your help on this part, the important change is we must have correct relative paths in our scss files.
Currently, we need to prefix with css which break other compilers and completion from ide